src/discord/commands/deploy.ts

```diff
@@ -0,0 +1,23 @@
+import { ApplicationCommandData, CommandInteraction } from 'discord.js';
+import commands from './index';
+
+export default {
+    name: 'deploy',
+    description: 'Deploys slash commands in current guild - meant for testing.',
+    options: [],
+
+    async execute(interaction: CommandInteraction): Promise<void> {
+        const data: ApplicationCommandData[] = [];
+
+        for (const command in commands) {
```

Comment:
A `for...of` loop here makes a bit more sense.

Reply:
That makes sense, I'll make the change.